src/main/java/nextstep/sessions/domain/SessionStudents.java

```diff
@@ -0,0 +1,38 @@
+package nextstep.sessions.domain;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class SessionStudents {
+
+    private Set<SessionStudent> students = new HashSet<>();
+
+    public SessionStudents() {
+        this.students = new HashSet<>();
+    }
+
+    public SessionStudents(List<SessionStudent> students) {
+        for (int i = 0; i < students.size() - 1; i++) {
+            checkStudent(students.get(i), students.get(i+1));
+        }
+        students.stream().forEach(student -> addStudent(student));
```

Comment:
```suggestion
        this.students = new HashSet<>(students);
```
생성자를 이용해서 바로 `Set` 구조로 변경해보는 건 어떨까요?

중복에 대한 검증은 `NsUser`의 equals, hashCode까지 구현하면(`SessionStudent`에서 `NsUser`를 이용해 equals를 구현하셨네요), `this.students`와 `students`의 크기가 동일한지 여부로도 확인이 가능할 것으로 보입니다.